changelogs/unreleased/gh-6990-type-of-CASE-operation.md

```diff
@@ -0,0 +1,3 @@
+## feature/sql
```

Comment:
In the docbot requests you are saying *"after this patch will be ..."*. Doc requests are not bound to a patch. They just happen to be in the commit messages. The text in doc requests should be the same like if you would create a doc ticket manually without talking about any patches. Also they are created after stuff is done, so the facts should be described like they have already happened. For example, not *"new rules will be applied"*, but *"new rules are applied"*.